modules/custom/activity_logger/activity_logger.tokens.inc

```diff
@@ -146,12 +151,18 @@ function activity_logger_tokens($type, $tokens, array $data, array $options, Bub
 
               // If it's a group.. add it in the arguments.
               if (isset($group) && $group instanceof Group) {
-                if ($name === 'gtitle') {
+                if ($name === 'gtitle' || $name === 'gtrimmed-title') {
                   $curr_langcode = \Drupal::languageManager()->getCurrentLanguage(LanguageInterface::TYPE_CONTENT)->getId();
+
                   if ($group->isTranslatable() && $group->hasTranslation($curr_langcode)) {
                     $group = $group->getTranslation($curr_langcode);
                   }
-                  $replacements[$original] = $group->label();
+
```

Comment:
@rollki Please add explanation comment on what we do here.

Reply:
Added description.